src/main/java/com/ngjackson/jegy/model/ticket/form/TicketForm.java

```diff
@@ -0,0 +1,52 @@
+package com.ngjackson.jegy.model.ticket.form;
+
+import com.ngjackson.jegy.model.form.Form;
+
+public class TicketForm extends Form {

```

Comment:
I would prefer the name `CreateTicketRequest` or `CreateUpdateTicketRequest`, but I'm not super passionate about that.